### PR TITLE
Added powershell as new langage with classes as first concept

### LINKED ISSUE #449: [PowerShell] Add PowerShell Core support
PowerShell is steadily increasing in popularity ever since going [cross-platform](https://github.com/PowerShell/PowerShell). The object-based pipeline makes it more reliable than ordinary text processing, and they've been adding more functions and operators with every major release. For what it still can't do with its default functionality, it can typically handle by falling back on .NET functions and methods.

Given Bash is already an option on Code Thesaurus, I'd really like to see PowerShell added as an option as well.

### LINKED ISSUE #694: [Language] Powershell
# Language you would like added

Powershell

# Information on the language

Official documentation : https://learn.microsoft.com/powershell/ 

# Disclaimer

Please can i work on this issue ?

### LINKED ISSUE #695: [Powershell] Add classes
# Description

Classes concept to add for Powershell.

# Requirements

No.

# Additional Notes

Command to use to generate the template : 
```bash
python .\manage.py generate_template --language-version 7 powershell classes
```

This issue should resolves ~#694~  #449  by adding Powershell as new langage in `web/thesauruses/meta_info.json` file.

### PR DESCRIPTION
**Pull request by ulyssear**
## What GitHub issue does this PR apply to?

Resolves #449 
Resolves #695

## What changed and why?

Powershell was missing in the available languages.

## (If editing Django app) Please add screenshots

   No screenshot is necessary.

## Checklist

- [ ] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes

## Any additional comments or things to be aware of while reviewing?

No.

### CODE REVIEW

File: web/thesauruses/powershell/7/classes.json [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    "extending_interface": {
+      "name": "Class/Interface that inherits/extends another class/interface",
+      "not-implemented": true
+    },
+    "calling_superclass_functions": {
+      "name": "Calling a superclass function",
+      "code": [
+        ""
+      ]
+    },
```
Comment (geekygirlsarah) [inline web/thesauruses/powershell/7/classes.json]: What's supposed to go here? Code or not-implemented?

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
Have one question, but then it's good to merge.

File: web/thesauruses/powershell/7/classes.json (hunk 1 above)
Comment (ulyssear) [reply web/thesauruses/powershell/7/classes.json]: No I missed this entry, It's not implemented in Powershell.

**Comment by ulyssear** [comment]
I know it's a bit late for this, is it possible to have the "hacktoberfest-accepted" label ?

**Review by geekygirlsarah — APPROVED** [review]
Thanks for the change. I'll merge it in.

**Pull request merged**